Source/SwiftLintFramework/Models/Configuration.swift

```diff
@@ -213,16 +213,11 @@ public struct Configuration {
             self.fileGraph = fileGraph
             setCached(forIdentifier: cacheIdentifier)
         } catch {
-            let errorString: String
-            switch error {
-            case let ConfigurationError.generic(message):
-                errorString = "SwiftLint Configuration Error: \(message)"
-
-            case let YamlParserError.yamlParsing(message):
-                errorString = "YML Parsing Error: \(message)"
-
-            default:
-                errorString = "Unknown Error"
+            guard let errorString = errorMessage(for: error, hasCustomConfigurationFiles: hasCustomConfigurationFiles)
+            else {
+                // Fall back to default
```

Comment:
Maybe `// Error isn't an actual error → silently fall back to default` or just `// Silently fall back to default`